File: Programming-languages-and-laboratory/Java/Exercises/28_compito_java_2019_02_21/Cliente.java
import java.io.*;
import java.util.*;

public class Cliente {

	private String nomeCognome;
	private List<Posteggio> posteggi;

	public Cliente(String nomeCognome) {
		this.nomeCognome = nomeCognome;
		this.posteggi = new LinkedList<Posteggio>();
	}

	public void addPosteggio(String targa, int giorni, double costo) {
		posteggi.add(new Posteggio(targa, giorni, costo));
	}

	public double getTot() {
		double tot = 0;
		for (Posteggio n : posteggi)
			tot = tot + n.getTot();
		return tot;

	}

	public String toString() {
		return nomeCognome + " " + getTot();
	}
}

File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/28_compito_Java_20190221/Cliente.py
class Cliente:
    def __init__(self, nome_cognome):
        self._nome_cognome = nome_cognome
        self._posteggi = []
        self._costo_totale = 0

    def add_costo_totale(self, c):
        self._costo_totale += c

    def add_posteggi(self, lp):
        self._posteggi.append(lp)

    def __str__(self):
        return self._nome_cognome + " " + str(self._costo_totale)
